components/common/icon-clutch/ClutchIcon.tsx

```diff
@@ -4,15 +4,13 @@ import SvgIcon, { type SvgIconProps } from '@mui/material/SvgIcon';
 
 const ClutchIcon: FC<SvgIconProps> = props => {
   return (
-    <SvgIcon {...props} viewBox="0 0 57 17">
+    <SvgIcon {...props} fill="currentColor" viewBox="0 0 57 17">
```

Comment:
As far as I know, `fill="currentColor"` comes from `SvgIconProps`. I believe it will work similarly if omitting a separate `fill` property. Could you double-check?